src/components/PipelineRunDetailsView/tabs/PipelineRunDetailsTab.tsx

```diff
@@ -203,17 +208,11 @@ const PipelineRunDetailsTab: React.FC<React.PropsWithChildren<PipelineRunDetails
                     </DescriptionListDescription>
                   </DescriptionListGroup>
                 )}
-                {sbomTaskRun && (
+                {imageDigest && (
                   <DescriptionListGroup>
                     <DescriptionListTerm>SBOM</DescriptionListTerm>
                     <DescriptionListDescription>
-                      <Link
-                        to={`/application-pipeline/workspaces/${workspace}/applications/${
-                          sbomTaskRun.metadata.labels[PipelineRunLabel.APPLICATION]
-                        }/taskruns/${sbomTaskRun.metadata.name}/logs`}
-                      >
-                        View SBOM
-                      </Link>
+                      <ExternalLink href={generateSbomUrl(imageDigest)}>View SBOM</ExternalLink>
```

Comment:
This should be conditionally rendered. 
If the Konflux administrator didn't configure a Bambino instance, we shouldn't show a link here.

Reply:
Yes, I agree, but how do we identify whether or not the bombino URL is available?

Reply:
Right now it's just a static URL and we append the image digest at the end of the url.

Reply:
Can we identify the community instance?

Reply:
yes, we can. do we want to send these notifications and show the SBOM URL only for internal instances?

Reply:
For now I would say yes. going forward I think that the integration of Konflux with an SBOM server will be valuable also for the community version. Thus, I think that the details of SBOM server should be read from a configmap on the cluster, but this can be done in a different PR in order to unblock this one.

Reply:
There isn't a proper way to identify the internal instance right now. We will figure this out in separate PR